src/fpsgame/fps.cpp

```diff
@@ -1062,6 +1065,7 @@ namespace game
 	/// render game hud depending on hudplayer's state and "SPECTATOR" in spectator mode
     void gameplayhud(int w, int h)
     {
+        return;
```

Comment:
that we do not render it at the moment, drawhudicons is pretty slow due to the new dynamic filepaths. This causes the bad fps while walking. #139 
We could implement it without lags (by loading to an static Texture and checking against its pointer) but i dont see the neccessarity of those icons anyways atm.